formats/shared/definitions/_specialist_document.jsonnet

```
@@ -169,6 +172,76 @@
       },
     },
   },
+  animal_disease_case_metadata: {
+    type: "object",
+    additionalProperties: false,
+    properties: {
+      bulk_published: {
+        type: "boolean",
+      },
+      disease_type: {
+        type: "array",
+          items: {
```

Tiny nitpick: I think the indentation of this line is wrong.

```suggestion
        items: {
```